frontend/src/citizen-frontend/income-statements/types/form.ts

```diff
@@ -109,6 +109,14 @@ export const empty: IncomeStatementForm = {
   assure: false
 }
 
+export const initialFormData = (
+  isValidStartDate: (date: LocalDate) => boolean
+): IncomeStatementForm => {
+  let startDate = LocalDate.today()
+  while (!isValidStartDate(startDate)) startDate = startDate.addDays(1)
+  return { ...empty, startDate: startDate.format() }
```

Comment:
I'm not sure if `max(startDates) + 1 day` is what we want 🤔  What we want is just some day in the future that doesn't exist in `startDates`.

Reply:
Doesn't the current implementation basically do that in most of the cases? It would be quite rare to have several income statements starting in the future. Maybe it could actually pick `max([...startDates, ...endDates]) + 1`?

Reply:
How about now? Doesn't take endDates in account, but uses `max(startDates) + 1 day`